src/jmcomic/jm_option.py

```
@@ -257,6 +257,14 @@ def decide_image_save_dir(self, photo: JmPhotoDetail, ensure_exists=True) -> str
             photo
         )
 
+        # 路径遍历防护：确保解析后的路径仍在 base_dir 范围内
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)